tests/stdlib/tmath.nim

```diff
@@ -1,13 +1,12 @@
 discard """
-  action: run
-  matrix:"; -d:nimTmathCase2 -d:danger --passc:-ffast-math"
+  targets: "c js"
```

Comment:
```suggestion
  targets: "c cpp js"
```

just to be on the safe side; cpp is usually redundant with c but math is important enough